src/pudl_archiver/archivers/eia/nrelatb.py

```diff
@@ -0,0 +1,33 @@
+"""Download NREL ATB for Electricity Parquet data."""
+
+from pathlib import Path
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+
+# Note: Using non s3:// link here as compatibility between asyncio and botocore is
+# complex.
+BASE_URL = "https://oedi-data-lake.s3.amazonaws.com/ATB/electricity/parquet/"
```

Comment:
Looking at the files in the draft archive, I see that they are each only 313 bytes in size, and contain an XML error message rather than any data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>ATB/electricity/parquet//2019/ATBe.parquet</Key><RequestId>PWTBPECAHFSA089K</RequestId><HostId>le0IwGW28lXyyxX+CENYiswSLVu9JszYLZguMcgrtq9vTaaDFTDTGCwyMAIYXgTdvfUFjgrr1EE=</HostId></Error>
```

Attempting to download from the HTTPS URL directly with this command works fine though:

```console
wget https://oedi-data-lake.s3.amazonaws.com/ATB/electricity/parquet/2019/ATBe.parquet
```

I think it might be the double slash `// after `BASE_URL` that's causing the problem.

Reply:
This PR isn't ready for review yet, I'm still testing out some other things to do with Parquet validation. But noted.

Reply:
This did indeed fix it! Parquet files are now downloading as expected.